packages/cli/src/oclif/commands/invoke.js

```diff
@@ -1378,6 +1379,10 @@ InvokeCommand.flags = buildFlags({
       description:
         'EXPERIMENTAL: Instead of using the local .env file, use the production authentication data with the given authentication ID (aka the "app connection" on Zapier). Find them at https://zapier.com/app/assets/connections (https://zpr.io/z8SjFTdnTFZ2 for instructions) or specify \'-\' to interactively select one from your available authentications. When specified, the code will still run locally, but all outgoing requests will be proxied through Zapier with the production auth data.',
     }),
+    'paging-token': Flags.string({
+      description:
+        'Set bundle.meta.paging_token. Used for search pagination or bulk reads. When used in production, this indicates which page of items you should fetch. First page is 0.',
```

Comment:
The "First page is 0." bit only applies to `bundle.meta.page`.

```suggestion
        'Set bundle.meta.paging_token. Used for search pagination or bulk reads. When used in production, this indicates which page of items you should fetch.',
```

Reply:
🤦‍♀️ Good catch. Cursor kept adding this, I missed it the last time before committing.